xqueue_watcher/settings.py

```diff
@@ -1,21 +1,10 @@
-import json
 
 MANAGER_CONFIG_DEFAULTS = {
     'HTTP_BASIC_AUTH': None,
     'POLL_TIME': 10,
     'REQUESTS_TIMEOUT': 1,
     'POLL_INTERVAL': 1,
+    'IDLE_POLL_INTERVAL': 60,
```

Comment:
I'm +1 with @stvstnfrd on this. It's wasteful, but xqueue for all of edx.org runs on a couple of t2.medium instances anyhow. I'd rather keep it stupid and minimize risk of regressions until we can take it out back and shoot it.

Reply:
@stvstnfrd @ormsbee fair enough. I'll keep it stupid, for now. :)  

Fun bit of history: I had an [ancient PR](https://github.com/edx/xqueue/pull/80) to make xqueue run in such a way that requests would block, rather than poll every second. I guess it never got merged because it wasn't a high priority at the time, and I stopped working for edX...

These days, I'd rewrite xqueue be an asyncio-based service, though that probably wouldn't get through code review, either 🤣 .